### ISSUE TITLE
Add a learnr site

### ISSUE THREAD

**Issue by kbenoit** [quanteda/quanteda#1513]
This looks awesome: https://rstudio.github.io/learnr/
Would be nice to integrate this with https://tutorials.quanteda.io.

**Comment by stefan-mueller** [comment]
Good idea, we could add exercises to https://tutorials.quanteda.io after each of the (currently) seven main chapters of the tutorial website. I could start working on this in January. 

@koheiw, is it possible to integrate **learnr** tutorials into the current layout of the website? I tried **learnr** and it seems that it only supports `.Rmd`, but our tutorial website uses `.Rmarkdown`.

**Comment by koheiw** [comment]
Worth experimenting. If it is a tool to insert small js, it should work with Rmakdown.
